meinberlin/apps/documents/templates/meinberlin_documents/chapter_detail.html

```diff
@@ -2,93 +2,58 @@
-                                {% endfor %}
-                            </ol>
-                        </div>
-                    </nav>
-                {% endif %}
-
-                <div class="chapter">
-                    <h2 class="chapter-title">{{ object.name }}</h2>
-
-                    {% for paragraph in object.paragraphs.all %}
-
-                        <section class="paragraph" id="paragraph-{{ object.pk }}-{{ paragraph.pk }}">
-                            <div class="paragraph__content">
-                                <h3 class="paragraph__title">
-                                    {{ paragraph.name }}
-                                </h3>
-                                <div class="ck-content">
-                                    {{ paragraph.text|richtext }}
-                                </div>
-                            </div>
-                            <div class="paragraph__actions">
-                                <a
-                                        class="btn btn--primary"
-                                        href="{{ paragraph.get_absolute_url }}"
-                                >
-                                    <i class="far fa-comment" aria-hidden="true"></i>
-                                    {% blocktranslate count count=paragraph.comments.all|length %}{{ count }} Comment{% plural %}{{ count }} Comments{% endblocktranslate %}
-                                </a>
+    <article class="u-spacer-top">
+        {% if chapter_list %}    
+            <div class="modul-toc">
+                <h2 class="title">{% translate 'Table of Contents' %}</h2>
+                <ol aria-label="{% translate 'Table of Contents' %}">
+                    {% for chapter in chapter_list %}
+                    <li>
+                        <a href="{{ chapter.get_absolute_url }}">
+                            {{ chapter.name }}
+                        </a>
+                    </li>
+                {% endfor %}
+                </ol>
+            </div>
+            <div>
+                <h2>{{ object.name }}</h2>
+                {% for paragraph in object.paragraphs.all %}
+                    <section id="paragraph-{{ object.pk }}-{{ paragraph.pk }}">
+                        <div>
+                            <h3>
+                                {{ paragraph.name }}
+                            </h3>
+                            <div class="ck-content">
+                                {{ paragraph.text|richtext }}
                             </div>
-                        </section>
-
-                    {% endfor %}
-                </div>
+                        </div>
+                        <div class="u-flex-end">
+                            <a href="{{ paragraph.get_absolute_url }}">
+                                <i class="far fa-comment" aria-hidden="true"></i>
+                                {% blocktranslate count count=paragraph.comments.all|length %}{{ count }}
```

Comment:
also in the design the format is `<Icon> Comments (<count>)`, here it's still `<Icon> <count> Comments`

Reply:
I'm fine with that as long as the process is not to nontransparent. If you make changes on the fly they should be reflected/changed in the designs (at least at a later point). Otherwise there's a constant disparity between the software and the designs and only you and marcelo know about it being intentional.